lib/charms/operator_libs_linux/v1/systemd.py

```diff
@@ -199,13 +141,13 @@ def service_pause(service_name: str) -> bool:
     Args:
         service_name: the name of the service to pause
     """
-    _systemctl("disable", service_name, now=True)
+    service_disable("--now", service_name)
     _systemctl("mask", service_name)
 
     if not service_running(service_name):
         return True
 
-    raise SystemdError("Attempted to pause '{}', but it is still running.".format(service_name))
+    raise SystemdError(f"Attempted to pause '{service_name}', but it is still running.")
```

Comment:
Probably won't matter here as I doubt service names have quotes in them, but `{service_name!r}` is better than `'{service_name}'` as it uses the repr (which includes quotes and escaping).

Reply:
Either way using `{service_name!r}` is good insurance that if someone decides to make the service names have quotes, it won't bork the charmlib and require us to issue another update. Let me make this change.

Reply:
Refactored in commit https://github.com/canonical/operator-libs-linux/pull/106/commits/af8c1fb3b34fbfc19b74b20d4210dd45c7eb07db